### DISCUSSION TITLE
I won't make destructive changes for a while

### DISCUSSION THREAD

**Discussion by mitosagi** [team-apm/apm#107] [General]
This is not an issue, so I'll write it here.

I will finish the destructive changes to the data structure with a PR that I have already submitted now. If you're stopping the release because you're worried that I'll make major changes, you can resume the release.

Please let me know if there is anything I can do to help you with the next release.

**Comment by hal-shu-sato** [comment]
Thank you for letting me know.
As well as breaking changes, it's just been hard to release recently because new features have been continuously developed and added every day.
I think the additions settled down yesterday, so I'm going to release apm as v1.0.0 this weekend.
Letting you know what will happen after the official release, I'll add a `develop` branch and we will continue to develop there.

What I would like to ask you to do before the release is to credit yourself with your name, as per #103.
Other than that, please keep doing what you've been doing. Since that would be very helpful...
Also, if you would like, you can become a collaborator of this repository.
I look forward to your reply.

**Comment by mitosagi** [comment]
I get it, I will continue with what I am doing now.

As for collaborators, I would like to be able to tag issues, assign them, and do other details myself, so I would appreciate it if you would consider it.

**Comment by hal-shu-sato** [comment]
Okay.
I'll send you the invitation for a collaborator.
Thank you for your cooperation.

**Comment by hal-shu-sato** [comment]
I forgot to send you the invitation for apm-data.
I sent it. Please check it.

**Comment by hal-shu-sato** [comment]
I decided not to use the `develop` branch.
I was going to run it with the default branch as `main`, but I found that troublesome for pull requests.
The source for the current version can be accessed via tags.
